### PR TITLE
Deprecate systems and resources internal namespaces

### PR DESCRIPTION
**Pull request by bigtimebuddy**
## Deprecated

* Removes the internal namespaces `systems` and `resources` in `@pixi/core`

### Context

This is a followup to discussion in https://github.com/pixijs/pixi.js/pull/6849#issuecomment-684932591. This change is consistent with #6681 and #6328 and has the added benefit of reducing our api-extractor complexity.

### 🔗  Links

Sanity Check: https://jsfiddle.net/bigtimebuddy/ukwsq86p/
Docs: https://pixijs.download/dev-resources-systems-namespaces/docs/index.html

### CODE REVIEW

**Review by ivanpopelyshev — APPROVED** [review]
Sooo many deprecations :)

**Comment by bigtimebuddy** [comment]
Yeah, I really just want to make a v6 release to remove all deprecations.

File: bundles/pixi.js/src/useDeprecated.ts [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
     BaseTexture,
     RenderTexture,
     Texture,
     State,
-    AbstractRenderer } from '@pixi/core';
+    AbstractRenderer,
+    StateSystem,
+    SVGResource,
+    ImageResource,
+    BaseImageResource } from '@pixi/core';
```
Comment (SukantPal) [inline bundles/pixi.js/src/useDeprecated.ts:14]: Shouldn't `} from '@pixi/core';` go on its own line?

File: packages/core/test/autoDetectResource.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
-const { resources } = require('../');
-const {
-    autoDetectResource,
+const { autoDetectResource,
     INSTALLED,
     CanvasResource,
     ImageResource,
     VideoResource,
-    SVGResource } = resources;
+    SVGResource } = require('../');
```
Comment (SukantPal) [inline packages/core/test/autoDetectResource.js:6]: Here, too. `} = require('../');` should be on its own line.

**Review by SukantPal — APPROVED** [review]
Optional suggestions included 😄

**Pull request merged**